src/main/java/org/terasology/core/world/CoreBiome.java

```diff
@@ -1,18 +1,6 @@
-/*
- * Copyright 2014 MovingBlocks
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Copyright 2021 The Terasology Foundation
```

Comment:
2014 -> 2021? What has changed? If nothing else in the file has changed then I might just leave the 2014 date in there.

Reply:
This is the case for most of the changes in this pull request.

Reply:
let's stick with our current practice of updating to the current year for now so we keep consistency
as mentioned on Discord, we can discuss our copyright practices in the next reviver meeting

@soloturn please update all copyright headers in hear to the current year 2023

Reply:
@jdrueckert i looked it up this time, and not updating would be the safe option:
https://techwhirl.com/updating-copyright-notices/

 “the year date [in the copyright notice] is more than one year later than the year in which publication first occurred, the work is considered to have been published without any notice….” 

that would mean, if somebody using code older than a year, it would have some default copyright and any committer could sue the person using it. of course one can argue that in git one sees that it was there before and had a different copyright. but for binary downloads? not ideal. @jdrueckert. i do agree with @BenjaminAmos  and reverted all to the original. but feel free to ping here in case you really discussed it in a reviewer meeting.